assets/src/edit-story/components/canvas/pagemenu/index.js

```diff
@@ -29,13 +29,13 @@ import { useCallback } from '@wordpress/element';
  */
 import { useStory, useHistory } from '../../../app';
 import { createPage } from '../../../elements';
-import Delete from './delete_icon.svg';
-import Duplicate from './duplicate_icon.svg';
-import Undo from './undo_icon.svg';
-import Redo from './redo_icon.svg';
-import Add from './add_page.svg';
-import Layout from './layout_helper.svg';
-import Text from './text_helper.svg';
+import Delete from '../../../icons/delete_icon.svg';
```

Comment:
Hmm. Not 100% sure on this. Some icons I can definitely see as universal. But some icons seem to be so tightly connected with a single use-case, that this would only create more distance between the resource and it's usage. WDYT?

Reply:
I will think about it @dvoytenko. I am divided between both approaches as well, but I definitely prefer to manage assets in one place to avoid redundancy, anyway, as you mentioned, we lost the local reference for local stuff. In general, what I generally use is aliases like `@assets/images`, do you think that can be useful for this scenario? cc @ndev1991

Reply:
I definitely like aliasing in webpack for assets. If we want centralized icons, that would be a good way to do it.

I'd do it in a separate PR though for clarity. Also there's some tooling setup - any opinions on this @swissspidy?

And remember you can even use [emoji aliases for added awesomeness](https://twitter.com/ryanflorence/status/1228749328837009410)! 😉

Reply:
As long as we don't start seeing things like:

```
icons/delete.svg
icons/delete_for_this_one_case.svg
icons/delete_for_this_other_case.svg
```

Even this is usually manageable, but IMHO more local placement is cleaner. But I don't have a very strong opinion on this. If there are other benefits from concentrating these resources, I can be persuaded.

Reply:
Happy to discuss such aliases in an issue, but in my experience they make it more difficult to grasp the code rather than to be actually helpful. But perhaps you can prove me wrong.

Reply:
My point was not to force such names. Actually quite the opposite. I was just reflecting that this could happen (and often does). E.g. there naturally might be a couple of very similar delete or duplicate buttons that are used in different situations. Having them co-located along the code clarifies where they are used and how, instead of ad-hoc giving them different names.